src/ecredis.erl

```diff
@@ -385,6 +385,10 @@ get_successes_and_retries(#query{response = {error, <<"TRYAGAIN">>}, retries = R
     {[], [Query#query{retries = Retries + 1}]};
 get_successes_and_retries(#query{response = {error, no_connection}} = Query) ->
     handle_error(no_connection, Query);
+get_successes_and_retries(#query{response = {error, gen_server_timeout}} = Query) ->
```

Comment:
I think no_connection and gen_server_timeout should be handled in a similar fashion. Let me know what you think.

Reply:
agree - what do we do now on no-connection?

Reply:
Basically they are handled the same way.

Reply:
We remap and try to find again where we should send the query